src/proto/Cardano.proto

```diff
@@ -0,0 +1,74 @@
+syntax = "proto3";
+
+package TW.Cardano.Proto;
+option java_package = "wallet.core.jni.proto";
+
+import "Common.proto";
+
+message OutPoint {
+    bytes tx_hash = 1;
+    uint64 output_index = 2;
+}
+
+message TxInput {
+    OutPoint out_point = 1;
+
+    string address = 2;
+
+    // ADA amount // TODO proper type
+    uint64 amount = 3;
+}
+
+message TxOutput {
+    string address = 1;
+    
+    // ADA amount // TODO proper type
+    uint64 amount = 2;
+}
+
+message Transfer {
```

Comment:
tss guys might need multiple outputs, let's support it

Reply:
We don't support that even for BTC (was done but discarded), it's a large change, I would not do it now for Cardano.